docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -0,0 +1,84 @@
+const sass = require('node-sass');
+const glob = require('glob');
+
+module.exports = function (grunt) {
+
+  function getIncludeFiles() {
+    const patterns = [
+      '**/*.scss',
+      'node_modules/decanter/scss',
+    ];
+
+    const libraries = [];
+    patterns.map(function (pattern) {
+      glob.sync(pattern).map(function (file) {
+        libraries.push(file);
+      });
+    });
+
+    return libraries;
+  }
+
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    watch: {
+      css: {
+        files: ['**/*.{scss,sass}'],
+        tasks: ['sass', 'run:stylelint'], // run stylelint at same time after each change
```

Comment:
running stylelint after every change is going to be slow, better to offload that task to an editor plugin and run stylelint in CI.

Reply:
ok. let's remove this from our grunt watch task and make a card to add sass-tests and linting to the CI for this repo